src/engine/source/builder/register.hpp

```diff
@@ -159,6 +160,9 @@ static void registerBuilders(std::shared_ptr<Registry> registry, const dependenc
     registry->registerBuilder(builders::opBuilderSpecificHLPQuotedParse, "helper.parse_quoted");
     registry->registerBuilder(builders::opBuilderSpecificHLPBetweenParse, "helper.parse_between");
     registry->registerBuilder(builders::opBuilderSpecificHLPAlphanumericParse, "helper.parse_alphanumeric");
+
+    // Update Confirmation
+    registry->registerBuilder(builders::opBuilderHelperSendUpgradeConfirmation, "helper.upgrade_confirmation_send");
```

Comment:
`upgrade_confirmation_send` sounds bad, can it be changed to `upgrade_send_confirmation` or `send_upgrade_confirmation`?

Reply:
will do